scripts/sign.py

```diff
@@ -16,43 +16,103 @@ def int_parse(str):
 
 def get_args():
     from argparse import ArgumentParser
-
-    parser = ArgumentParser()
+    command_choices = ['sign', 'generate-digest', 'stitch-ta']
+
+    parser = ArgumentParser(
+        description='Sign a Tusted Application for OP-TEE.',
+        usage='\n   %(prog)s command [ command_opts ] command_args\n\n'
+
+        '   command:\n' +
+        '     sign             Generate signed loadable TA image file.\n' +
+        '                      Takes arguments --uuid, --in, --out' +
+        ' and --key.\n' +
+        '     generate-digest  Generate loadable TA binary image digest' +
+        ' for offline\n' +
+        '                      signing. Takes arguments  --uuid, --in and' +
+        ' --out.\n' +
+        '     stitch-ta        Generate loadable signed TA binary image' +
+        ' file from\n' +
+        '                      TA raw image and its signature. Takes' +
+        ' arguments\n' +
+        '                      --uuid, --in, --out, and --sig.\n\n' +
+
+        'command_opts\n' +
+        '    --version NUM    NUM shall be 0. Defaults to 0.\n\n' +
```

Comment:
This argument could be removed.

Reply:
Done.

Reply:
Also did the required update of the linker scripts.